src/main/scala/com/rbmhtechnology/eventuate/crdt/LWWRegister.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2015 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.eventuate.crdt
+
+import akka.actor.ActorRef
+import akka.actor.ActorSystem
+import com.rbmhtechnology.eventuate.{Versioned, DurableEvent, VectorTime}
+
+import scala.concurrent.Future
+
+/**
+ * A LWWVersioned value.
+ *
+ * @param value The value.
+ * @param vectorTimestamp Timestamp of the event that caused this version.
+ * @param wallTimestamp Wall Clock
+ * @param creator Creator of the event that caused this version.
+ */
+case class LWWVersioned[A](value: A, vectorTimestamp: VectorTime, wallTimestamp: Long = 0L, creator: String = "")
```

Comment:
- Rename `wallTimestamp` to `systemTimestamp` to be consistent with naming used elsewhere?
- Maybe `creator` can also be renamed to `emitterId` since `LWWVersioned` is only used here?